src/days-2-4-adv/adv.py

```diff
@@ -35,6 +36,14 @@
 room['narrow'].n_to = room['treasure']
 room['treasure'].s_to = room['narrow']
 
+#items
+item = {
+    'dagger': Item("Dagger",
+    "Pointy end goes in target")
```

Comment:
This is a clever description; it's simple things like this that really contributes to user experience!